include/xbee/ble_frames.h

```diff
@@ -0,0 +1,120 @@
+#define XBEE_BLE_SCAN_VALUES_MAX_USEC       (0x270FD8F)
+#define XBEE_BLE_SCAN_VALUES_MIN_USEC       (0x9C4)
+
+/// Frame Type: BLE Scan Request
+#define XBEE_FRAME_BLE_SCAN_REQUEST             0x34
+/// Format of the XBee API frame type 0x34 (#XBEE_FRAME_BLE_SCAN_REQUEST)
+/// Sent from host to XBee, which responds with XBEE_FRAME_BLE_SCAN_STATUS
+typedef XBEE_PACKED(xbee_frame_ble_scan_req_t, {
+    uint8_t    frame_type;          ///< XBEE_FRAME_BLE_SCAN_REQUEST (0x34)
+    uint8_t    start_cmd;           ///< start/stop command, start=1 & stop=0
+    uint16_t   scan_duration_be;    ///< scan duration in seconds
+    uint32_t   scan_window_be;      ///< scanning window in microseconds
+    uint32_t   scan_interval_be;    ///< scanning interval in microseconds
+    uint8_t    filter_type;         ///< type of filters to apply to scan
+#define XBEE_BLE_SCAN_FILTER_NONE         ( 0 )
+#define XBEE_BLE_SCAN_FILTER_NAME         ( 1 << 0 )   // Filter on local name. 
+    uint8_t    filter[1];           ///< filter value, max 22 bytes
+}) xbee_frame_ble_scan_req_t;
+
+typedef XBEE_PACKED(xbee_header_ble_scan_req_t, {
+    uint8_t    frame_type;          ///< XBEE_FRAME_BLE_SCAN_REQUEST (0x34)
+    uint8_t    start_cmd;           ///< start/stop command, start=1 & stop=0
+    uint16_t   scan_duration_be;    ///< scan duration in seconds
+    uint32_t   scan_window_be;      ///< scanning window in microseconds
+    uint32_t   scan_interval_be;    ///< scanning interval in microseconds
+    uint8_t    filter_type;         ///< type of filters to apply to scan
+}) xbee_header_ble_scan_req_t;
+
+/// Frame Type: BLE Scan Status
+#define XBEE_FRAME_BLE_SCAN_STATUS              0xB5
+/// Format of the XBee API frame type 0xB5 (#XBEE_FRAME_BLE_SCAN_STATUS)
+/// Sent from XBee to host.
+typedef XBEE_PACKED(xbee_frame_ble_scan_status_t, {
+    uint8_t    frame_type;         ///< XBEE_FRAME_BLE_SCAN_STATUS (0xB5)
+    uint8_t    scan_status;        ///< request status
+#define XBEE_BLE_SCAN_STATUS_SUCCESS            0x00
+#define XBEE_BLE_SCAN_STATUS_RUNNING            0x01
+#define XBEE_BLE_SCAN_STATUS_STOPPED            0x02
+#define XBEE_BLE_SCAN_STATUS_ERROR              0x03
+#define XBEE_BLE_SCAN_STATUS_INVALID_PARAMS     0x04
+}) xbee_frame_ble_scan_status_t;
+
+/// Frame Type: BLE Scan Legacy Advertisement Response
+#define XBEE_FRAME_BLE_SCAN_LEGACY_ADVERT_RESP  0xB4
+/// Sent from XBee to host. Sent out when an advertisement is found.
+typedef XBEE_PACKED(xbee_frame_ble_scan_legacy_advert_resp_t, {
+    uint8_t    frame_type;         ///< XBEE_FRAME_BLE_SCAN_LEGACY_ADVERT_RESP (0xB4)
+    uint8_t    address[6];         ///< BLE MAC address of received advertisment
+    uint8_t    address_type;       ///< indicates the address type, public or random
+    uint8_t    advert_flags;       ///< advertisement flags
+    int8_t     rssi;               ///< receieve signal strength, in dBm
```

Comment:
Based on the sample, it appears that the `rssi` field will always be positive, but represents a negative value.  Should we declare it `uint8_t`?  At the very least, mention that it will be something like "0 to 127, representing 0 to -127dBm".
(Also, I just saw the misspelling of `received` in that comment.)

Reply:
Based on checking the XBee 3 BLU code, and the SiLabs Bluetooth API, it is indeed a `int8_t` (signed), -127 to +20.